storage/flux/reader.go

```diff
@@ -619,6 +620,7 @@ func (wai *windowAggregateIterator) Do(f func(flux.Table) error) error {
 
 const (
 	CountKind = "count"
+	SumKind   = "sum"
```

Comment:
oh, was sum not available before? Or are you just using it for the first time in this file?

Reply:
I used it for the first time in a test file, and storage can't import the universe package, so I needed to add it here.